extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboardOverviewPage.ts

```diff
@@ -263,13 +263,13 @@ function createServiceEndpointRow(modelBuilder: azdata.ModelBuilder, container:
 				azdata.connection.openConnectionDialog(undefined, connProfile);
 			}
 		});
-		endPointRow.addItem(endpointCell, { CSSStyles: { 'width': `${serviceEndpointRowEndpointCellWidth}px`, 'min-width': `${serviceEndpointRowEndpointCellWidth}px`, 'overflow': 'hidden' } });
+		endPointRow.addItem(endpointCell, { CSSStyles: { 'width': `${serviceEndpointRowEndpointCellWidth}px`, 'min-width': `${serviceEndpointRowEndpointCellWidth}px` } });
 	}
 	else {
 		const endpointCell = modelBuilder.text()
-			.withProperties({ value: endpoint.endpoint, CSSStyles: { ...cssStyles.text } })
+			.withProperties({ value: endpoint.endpoint, CSSStyles: { 'overflow': 'hidden', 'text-overflow': 'ellipsis', ...cssStyles.text } })
```

Comment:
Whenever we truncate and show ellipsis, it would be good to show tooltip on hover, which shows the entire string (or atleast most of it).